packages/core/useElementSize/index.ts

```diff
@@ -38,9 +38,9 @@ export function useElementSize(
       if (window && isSVG.value) {
         const $elem = unrefElement(target)
         if ($elem) {
-          const styles = window.getComputedStyle($elem)
-          width.value = Number.parseFloat(styles.width)
-          height.value = Number.parseFloat(styles.height)
+          const rect = $elem.getBoundingClientRect()
+          width.value = Number.parseFloat(rect.width)
```

Comment:
The types of (rect. width and rect. height) are already numbers, so there is no need to convert them using number. parseFloat